fiftyone/core/config.py

```diff
@@ -651,3 +651,20 @@ def _get_installed_packages():
     except:
         logger.debug("Failed to get installed packages")
         return set()
+
+
+class HTTPRetryConfig(object):
+    """Values used to configure the behavior of the retry logic
+    of HTTP calls made throughout the app
+    """
+
+    # HTTP codes that should trigger a retry
+    RETRY_CODES = {408, 429, 500, 502, 503, 504, 509}
```

Comment:
wouldn't the 500 codes be errors that won't fix themselves with retries?

Reply:
Yeah, this is confusing -- the logic in the decorator/library that uses these codes implements a negation:
```
giveup=lambda e: hasattr(e, "error_code")
        and e.error_code not in HTTPRetryConfig.RETRY_CODES,```

Reply:
Then could we change the comment to say `HTTP codes that will not trigger a retry`?  That would help clarify things the next time I say dumb stuff 😝 

This makes sense to me.

Reply:
That is absolutely true. I'm the one that wrote that original comment and I had my understanding backwards when I wrote it :D

Reply:
And I copied them from what Brian had originally made :grin:  -- There was no small amount of discussion around these codes; they were mostly lifted from the various cloud service documentations